site/en/r2/tutorials/distribute/keras.ipynb

```diff
@@ -319,102 +362,103 @@
         "      tf.keras.layers.Dense(64, activation='relu'),\n",
         "      tf.keras.layers.Dense(10, activation='softmax')\n",
         "  ])\n",
-        "\n",
+        "  \n",
         "  model.compile(loss='sparse_categorical_crossentropy',\n",
-        "                optimizer=tf.keras.optimizers.Adam(),\n",
+        "                optimizer=tf.keras.optimizers.Adam())\n",
         "                metrics=['accuracy'])"
-      ]
+      ],
+      "execution_count": 0,
+      "outputs": []
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "8i6OU5W9Vy2u"
       },
+      "cell_type": "markdown",
       "source": [
         "## Define the callbacks.\n",
         "\n"
       ]
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "YOXO5nvvK3US"
       },
+      "cell_type": "markdown",
       "source": [
         "The callbacks used here are:\n",
         "\n",
         "*   *Tensorboard*: This callback writes a log for Tensorboard which allows you to visualize the graphs.\n",
         "*   *Model Checkpoint*: This callback saves the model after every epoch.\n",
         "*   *Learning Rate Scheduler*: Using this callback, you can schedule the learning rate to change after every epoch/batch.\n",
-        "\n"
+        "\n",
+        "For illustrative purposes, we're also adding a print callback to display the Learning Rate in the notebook."
```

Comment:
FYI, Google Dev Docs style is to prefer second person: https://developers.google.com/style/person

Rest looks good. Thanks!